en/docs/learn/analytics/managing-dashboard-permissions.md

```diff
@@ -38,6 +133,8 @@ Follow the instructions below to change the default permissions that are set for
 
 ## Permission level and user role mapping
 
+As described above the permission level of a logged-in user is decided based on the available scopes. The logged-in user possesses a user role, and scopes are mapped to that user role. You can find the mapping between default scopes and default user roles in the [role mapping table]({{base_path}}/administer/managing-users-and-roles/managing-user-roles/#adding-role-mappings).
```

Comment:
based on the assigned scopes?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1697/commits/ab12d89bb02874fd1eccd6b48d0a387a8ccd7ffa